src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -515,8 +542,15 @@ def info(self):
                 'earmark': earmark,
                 'normalization': normalization,
                 'casesensitive': casesensitive,
+                'source': 'N/A'
```

Comment:
@rishabh-d-dave correct me if I am wrong - this will still show NA (or maybe `{}`) in the JSON output for non-clones, yes?

Reply:
Yes, this does show `source: 'N/A'` for subvolume which isn't a clone. We have to rely on the `type` of the subvolume to fix this. There is no other differentiator between clones with source information deleted from .meta and clones with source info retained.

Reply:
Right. We need to have that fix.

Reply:
Fixed.